<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="0" file="test_valid_fail_inst.py" name="test_valid_fail_inst.TestValidFailInst-20200407091618" skipped="0" tests="1" time="0.000" timestamp="2020-04-07T09:16:18">
	<testcase classname="test_valid_fail_inst.TestValidFailInst" file="spec/python/test_valid_fail_inst.py" line="9" name="test_valid_fail_inst" time="0.000" timestamp="2020-04-07T09:16:18">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
